site/docs/spec.md

```diff
@@ -333,6 +333,38 @@ Table metadata is stored as JSON. Each table metadata change creates a new table
 
 The atomic operation used to commit metadata depends on how tables are tracked and is not standardized by this spec. See the sections below for examples.
 
+### Delete Format
+
+This section details how to encode row-level deletes in Iceberg metadata. Row-level deletes are not supported in the current format version, 1. This part of the spec is not yet complete and will be completed as format version 2.
+
+#### Position-based Delete Files
+
+Position-based delete files identify rows in one or more data files that have been deleted. It has the schema as following:
+```json
+{
+  "type": "struct",
+  "fields": [ {
+    "id": 1,
+    "name": "file_path",
+    "required": true,
+    "type": "string",
+    "doc": "The full URI of a data file, with FS scheme. This must match the file_path of the target data file in a manifest entry."
```

Comment:
I don't think that requirements for these fields are obvious enough in docs. Let's use the doc field to describe what it is, but the spec requirements should be pulled out and clear.